code/game/gamemodes/miniantags/revenant/revenant_abilities.dm

```diff
@@ -343,16 +343,6 @@
 		if(iscarbon(mob))
 			if(ishuman(mob))
 				var/mob/living/carbon/human/H = mob
-				if(H.dna && H.dna.species)
-					H.dna.species.handle_mutant_bodyparts(H,"#1d2953")
-					H.dna.species.handle_hair(H,"#1d2953")
-					var/old_color = H.color
-					H.color = "#1d2953"
-					spawn(20)
-						if(H && H.dna && H.dna.species)
-							H.dna.species.handle_mutant_bodyparts(H)
-							H.dna.species.handle_hair(H)
-							H.color = old_color
```

Comment:
this is conflicting with the blight virus below, if we keep this short temporary effect it will remove the blight virus' color when in stage 5 even if not cured yet. Another solution would be to put the two on different priorities but I don't want blight on admin priority or the temp effect on washable priority, and I don't to add a fifth priority just for this.